config_prep.ipynb

```diff
@@ -158,6 +158,7 @@
     "def format_yaml_pe(RUN, samples_pe,\n",
     "                   TMP_DIR_ROOT = '/localscratch',\n",
     "                   host_db = '/home/jgsanders/ref_data/genomes/Homo_sapiens_Bowtie2_v0.1/Homo_sapiens',\n",
+    "                   bloom_db = '/home/a3tripat/share/kd_dbs/bloom',\n",
```

Comment:
is ```/home/a3tripat/share/kd_dbs/bloom``` the folder, or the folder + the basename for the indexed file? It should be the latter. 

Also, would be helpful to make read-access for other knightlabbers.